docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,253 +8,313 @@ weight: 200
 
-**Grafana Enterprise**: `grafana/grafana-enterprise`
+Grafana Docker images come in two editions:
+- **Grafana Enterprise**: `grafana/grafana-enterprise`
+- **Grafana Open Source**: `grafana/grafana-oss`
 
-**Grafana Open Source**: `grafana/grafana-oss`
+> **Note:** The recommended and default edition of Grafana is Grafana Enterprise. It is free and includes all the features of the OSS edition. Additionally, you have the option to upgrade to the [full Enterprise feature set](/products/enterprise/?utm_source=grafana-install-page), which includes support for [Enterprise plugins](/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
 
-Each edition is available in two variants: Alpine and Ubuntu. See below.
+The default images for Grafana are created using the widely used Alpine Linux project and can be found in the Alpine official image. For instructions on how to configure a docker image for Grafana, refer to [configure a Grafana Docker image](https://grafana.com/docs/grafana/latest/administration/configure-docker/).
 
-For documentation regarding the configuration of a docker image, refer to [configure a Grafana Docker image](https://grafana.com/docs/grafana/latest/administration/configure-docker/).
+## Run Grafana via Docker CLI
 
-This topic also contains important information about [migrating from earlier Docker image versions](#migrate-from-previous-docker-containers-versions).
+This section shows you how to run Grafana using the Docker CLI. 
 
-## Alpine image (recommended)
+> **Note:** If you are on a Linux system (for example, Debian or Ubuntu), you might need to add **sudo** before the command or add your user to the **docker** group. For more information, refer to [Linux post-installation steps for Docker Engine](https://docs.docker.com/engine/install/linux-postinstall/).
 
-**Grafana Enterprise edition**: `grafana/grafana-enterprise:<version>`
+To run the latest stable version of Grafana, run the following command:
 
-**Grafana Open Source edition**: `grafana/grafana-oss:<version>`
+```bash
+docker run -d -p 3000:3000 --name=grafana grafana/grafana-enterprise
+```
 
-The default images are based on the popular [Alpine Linux project](http://alpinelinux.org), available in [the Alpine official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images, and thus leads to slimmer and more secure images.
+where:
 
-The Alpine variant is highly recommended when security and final image size being as small as possible is desired. The main caveat to note is that it uses [musl libc](http://www.musl-libc.org) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software don't have an issue with this, so this variant is usually a very safe choice.
+run = run directly from the command line\
```

Comment:
Why is there a \ at the end of each line? Wondering if they are necessary.

Reply:
The \ was to add a line break (new-line) as in my preview it was all showing in a single line and looked unreadable. Please remove it if required.

Reply:
done